gx_suite_definitions/model_info.ipynb

```diff
@@ -0,0 +1,168 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "91f352ec",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "import great_expectations as gx\n",
+    "import synapseclient\n",
+    "\n",
+    "from agoradatatools.gx import GreatExpectationsRunner\n",
+    "\n",
+    "context = gx.get_context(project_root_dir='../src/agoradatatools/great_expectations')\n",
+    "\n",
+    "from expectations.expect_column_nested_field_values_mostly_regex import (\n",
+    "    ExpectColumnNestedObjectRegexRule,\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e17078fa",
+   "metadata": {},
+   "source": [
+    "# Create Expectation Suite for UI Config Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e75ee319",
+   "metadata": {},
+   "source": [
+    "## Get Example Data File"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "202f1ebe",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "syn = synapseclient.Synapse()\n",
+    "syn.login()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "4a4cfa90",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_details = syn.get(\"syn66330545\").path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e187fccf",
+   "metadata": {},
+   "source": [
+    "## Create Validator Object on Data File"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "08ef848d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df = pd.read_json(model_details)\n",
+    "nested_columns = [\"genetic_info\", \"biomarkers\", \"pathology\"]\n",
+    "df = GreatExpectationsRunner.convert_nested_columns_to_json(df, nested_columns)\n",
+    "validator = context.sources.pandas_default.read_dataframe(df)\n",
+    "validator.expectation_suite_name = \"model_details\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7cbd5113",
+   "metadata": {},
+   "source": [
+    "## Add Expectations to Validator Object For Each Column"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0c220329",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "validator.expect_column_nested_object_regex_rule(column=\"genetic_info\", target_field=\"ensembl_gene_id\", regex_pattern=\"^ENSMUSG\", valid_threshold=0.4)"
```

Comment:
I am not sure what threshold to use. But based on the result, currently there are 0.49 of valid string based on this regex pattern: `^ENSMUSG`

Reply:
I think the regex pattern needs to match `^ENSMUSG` _or_ `^ENSG` because we have both human and mouse IDs as possible values. I _think_ the regex would then be `^ENS(MUS)?G`?
 
For this one I would assume the proportion of entries with a valid Ensembl ID should be exactly 100%, not "mostly"... @JessterB are there any cases where `ensembl_gene_id` is empty/None?

Reply:
@jaclynbeck-sage You are correct that we do expect 100% to have values. I guess I was thinking we would write a 'mostly regex' rule for each kind of ensembl_gene_id, i.e. one that starts with ENSG and is exactly 15 characters vs one that starts with ENSMUSG and is exactly 18 characters. I want to use a more robust regex than just checking the prefix, given that I manually populated the source data and I don't trust me. It's probably possible to write a single regex that covers both cases, but I am bad at regexes.

@linglp As an example implementation this is just fine. Once we finalize our MVP data, we'll know how many ENSG vs ENSMUSG values we expect and can adjust the regex(es) and/or the threshold(s) as required.